MCSync/Syncer.cs

```diff
@@ -72,6 +72,19 @@ private async System.Threading.Tasks.Task DoSync()
 
                 List<Task> tasks = new List<Task>();
 
+                // Create Required Subdirectories
+                try
+                {
+                    Directory.CreateDirectory(modsDirectory);
+                    Directory.CreateDirectory(resourcePackDirectory);
+                    Directory.CreateDirectory(configFilesDirectory);
+                    Directory.CreateDirectory(oResourcesDirectory);
+                }
+                catch(IOException iox)
+                {
+                    Task.Errors.Add($"An unexpected error occurred during Sync: {iox.Message}");

```

Comment:
I don't think we even need to put this in a try/catch. The outer try/catch will do exactly this

Reply:
Fixed